python/ray/serve/tests/test_standalone2.py

```diff
@@ -604,15 +610,27 @@ def deployment_running():
         wait_for_condition(deployment_running, timeout=15)
         pid1 = requests.get("http://localhost:8000/f").text
 
-        updated_options = {
-            "num_replicas": 2,
-            "autoscaling_config": {"max_replicas": 2},
-            "user_config": {"name": "bob"},
-            "ray_actor_options": {"num_cpus": 0.2},
-        }
-        config_template["deployments"][0][option_to_update] = updated_options[
-            option_to_update
-        ]
+        if field_to_update == "import_path":
+            config_template[
+                "import_path"
+            ] = "ray.serve.tests.test_config_files.pid.bnode"
+        elif field_to_update == "runtime_env":
+            config_template["runtime_env"] = {
+                "working_dir": (
+                    "https://github.com/ray-project/test_dag/archive/"
```

Comment:
Instead of using a remote `working_dir`, can we simply change an `env_var`? For example,

```python
config_template["runtime_env"] = {"env_vars": {"test_var": "test_val"}}
```